ui-packages/packages/management-console-webapp/package.json

```
@@ -19,7 +19,7 @@
     "dev:restServer": "nodemon ./server/restServer.js",
     "dev:server": "nodemon ./server/app.js",
     "dev": "concurrently 'yarn  start' 'yarn run dev:server'",
-    "dev-remote-dataindex": "yarn start --define process.env.KOGITO_DATAINDEX_HTTP_URL='\"http://localhost:8180/graphql\"'",
+    "dev-remote-dataindex": "yarn start --env KOGITO_DATAINDEX_HTTP_URL='\"http://localhost:8180/graphql\"'",
```

```suggestion
    "dev-remote-dataindex": "yarn start --env development",
```
why don't we use a `development` or `dev` environment and then we create a `.env.development` file containing
```
NODE_ENV=development
KOGITO_DATAINDEX_HTTP_URL=http://localhost:8180/graphql
```
would it be possible? or even to https://github.com/mrsteele/dotenv-webpack foe the different webpack configurations.

wdyt?

---

```suggestion
    "dev-remote-dataindex": "yarn start --env KOGITO_DATAINDEX_HTTP_URL='http://localhost:8180/graphql'",
```

@AjayJagan I think you must remove the `\"` to make it work

---

Thanks! Pere